pygotham/frontend/sponsors.py

```diff
@@ -77,7 +77,9 @@ def edit(pk):
 def index():
     """Return the sponsors."""
     levels = Level.query.current.order_by(Level.order)
-    return render_template('sponsors/index.html', levels=levels)
+    has_sponsors = any(l.accepted_sponsors.count() for l in levels)
```

Comment:
A nitpick, but could we do something along the lines of

```python
db.session.query(Sponsor.query.filter(
    Level.event == g.current_event,
    Sponsor.accepted == True,
).exists()).scalar()
```
instead? As written, this will make at most the as many queries as there are sponsorship levels. Querying the `Sponsor` model more closely matches the question we're asking and limits the operation to a single query.